src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Main class
+ */
+package Duke;
+
+import Command.Command;
+
+import java.time.format.DateTimeParseException;
+
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructor that initialises the relative file path to save / load
+     * the specific file
+     *
+     * @param filePath which contains the relative path of the saved task list
+     */
+    public Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        tasks = new TaskList(storage.readFile());
+    }
```

Comment:
hmm try-catch may be better here cos must handle case where filePath may not exist

Reply:
Oh okay! Thanks for the suggestion :)